custom_components/hacs/manifest.json

```diff
@@ -16,11 +16,11 @@
         "lovelace",
         "repairs"
     ],
-    "documentation": "https://hacs.xyz/docs/configuration/start",
+    "documentation": "https://hacs.xyz/docs/use/configuration/options",
```

Comment:
This is not only used for the option flow.
This should probably just be `https://hacs.xyz/docs/use/`

Reply:
Updated!